apps/ceramic/src/deploy-models.ts

```diff
@@ -0,0 +1,61 @@
+import { CeramicClient } from "@ceramicnetwork/http-client";
+import { DID } from "dids";
+import { Ed25519Provider } from "key-did-provider-ed25519";
+import { getResolver } from "key-did-resolver";
+import { fromString } from "uint8arrays/from-string";
+import * as fs from "node:fs";
+import * as path from "node:path";
+
+import { Composite } from "@composedb/devtools";
+import {
+  createComposite,
+  writeEncodedComposite,
+} from "@composedb/devtools-node";
+
+import "dotenv/config";
+
+export async function createAndDeploy() {
+  const privateKey = fromString(process.env.DID_PRIVATE_KEY as string, "hex");
+
+  const did = new DID({
+    resolver: getResolver(),
+    provider: new Ed25519Provider(privateKey),
+  });
+
+  await did.authenticate();
+
+  const ceramic = new CeramicClient(process.env.CERAMIC_API_URL);
+  ceramic.did = did;
+
+  const composites: Composite[] = [];
+
+  const filePaths = fs.readdirSync(path.resolve("src/models"));
```

Comment:
maybe we could try to utilise some existing migration tool for handling this, wdyt?

Reply:
Migration tools are usually targeting one or more storage engines. Afaik there is not tool specifically for ComposeDB.

Reply:
We could potentially store our "migrations" in the repo. Maybe we can keep it like this for now